app/javascript/packages/document-capture/components/po-search-fallback-select.scss

```diff
@@ -0,0 +1,3 @@
+.usa-input--error.usa-input--error.usa-input--error {
```

Comment:
`po-search-state-select` seems more descriptive here to me as a filename

Reply:
changed to `select-error.scss`